src/aws-lambda/personalize-delete-resources/bundle.sh

```diff
@@ -16,7 +17,12 @@ cd package
 zip -r9 ${OLDPWD}/${PACKAGE_FILE} .
 cd ${OLDPWD}
 
+echo "Downloading Personalize SDK model files"
+wget -q https://raw.githubusercontent.com/boto/botocore/e22aad94ba9f33bda7a0abea3a959fb1af56b25d/botocore/data/personalize-runtime/2018-05-22/service-2.json -P ./models/personalize-runtime/2018-05-22
```

Comment:
Are we sure these URLS are permanent? Shouldn't it be something like
https://raw.githubusercontent.com/boto/botocore/master/botocore/data/personalize/2018-05-22/service-2.json

(with the right branch and version 2018-05-22 )  ?

or include the json in the repo if we want to fix a version?